src/libcollections/string.rs

```diff
@@ -964,7 +961,7 @@ pub fn as_string<'a>(x: &'a str) -> DerefString<'a> {
     DerefString { x: as_vec(x.as_bytes()) }
 }
 
-#[unstable(feature = "collections", reason = "associated error type may change")]
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
Hm ok, I've switched it over to an `#[unstable]` `ParseError` newtype in the module.